articles/azure-functions/durable-functions-disaster-recovery-geo-distribution.md

```diff
@@ -0,0 +1,81 @@
+
+![Diagram showing scenario 2.](media/durable-functions-disaster-recovery-geo-distribution/durable-functions-geo-scenario02.png)
+
+This approach adds improvements on the previous scenario:
+- In the case of failure, failover to the failover regions is still managed by Traffic Manager. However, because the FunctionApp relies on its own storage account, the durable functions continue to work.
+- During a failover, there is no additional latency in the failover region, since the FunctionApp and the storage account are co-located.
+- Failure of the storage layer will cause failures in the durable functions, which, in turn, will trigger a redirection to the failover region. Again, since the FunctionApp and storage are isolated per region, the durable functions will continue to work.
+ 
+Important considerations for this scenario:
+- If the Function App is deployed using a dedicated AppService plan, there is an increased cost by replicating the compute instance in the failover datacenter.
+- Current state is not failed over. This implies that executing and checkpointed functions will fail. It is up to the client application to retry/restart the work.
+
+## Scenario 3 - Load Balanced Compute with GRS Shared Storage
+This scenario is a modification over the first scenario, implementing a shared storage account. The main difference that the storage account is created with geo-replication enabled.
+Functionally, this scenario provides the same advantages as Scenario 1, but it enables additional data recovery advantages:
+- Geo-redundant storage (GRS) and Read-access GRS (RA-GRS) maximize availability for your storage account. Additionally, RA-GRS provides read-only access to the data in the secondary location, in addition to geo-replication across two regions.
+- If there is a region outage of the storage service, one of the possibilities is that the datacenter operations determine that storage must be failed over to the secondary region. In this case, storage account access will be redirected transparently to the geo-replicated copy of the storage account, without user intervention.
+- In this case, state of the durable functions will be preserved up to the last replication of the storage account.
+
+As with the other scenarios, there are important considerations:
+- Failover to the replica is done by datacenter operators and it may take some time. Until that time, the FunctionApp will suffer an outage.
+- There is an increased cost for using geo-replicated storage accounts.
+- GRS occurs asynchronously. It is possible that some or the latest transactions are lost because of the latency of the replication process.
+
+You can read more about [Designing Highly Available Applications using RA-GRS](https://docs.microsoft.com/en-us/azure/storage/common/storage-designing-ha-apps-with-ragrs)
+
+![Diagram showing scenario 3.](media/durable-functions-disaster-recovery-geo-distribution/durable-functions-geo-scenario03.png)
```

Comment:
At end need `## Next steps` section with "For more information, see ..." links

for example, the scenarios talk about configuring traffic manager, so it would be good to link to the documentation that explains how to do that.

Reply:
check